lookout/templates/lookout-deployment.yaml

```diff
@@ -29,6 +29,14 @@ spec:
             items:
               - key: config.yaml
                 path: config.yaml
+        {{- if .Values.providers.github.secretName }}
+        - name: lookout-github-key-volume
+          secret:
+            secretName: {{ Values.providers.github.secretName }}
+            items:
+              - key:  {{ Values.providers.github.private_key }}
```

Comment:
Please make this a requirement or someone will forget to add it

Reply:
Done